Documentation/gettingstarted/kind.rst

```diff
@@ -154,51 +154,8 @@ We can now create the respective clusters:
     kind create cluster --name=cluster1 --config=kind-cluster1.yaml
     kind create cluster --name=cluster2 --config=kind-cluster2.yaml
 
-Deploy Cilium
--------------
-
-This is the same helm command as from :ref:`kind_install_cilium`. However
-we're enabling managed etcd and setting both ``cluster-name`` and
-``cluster-id`` for each cluster.
```

Comment:
I take it this part is not longer required?

Reply:
They are required - the clustermesh guide also has  these instructions.

Reply:
(From https://github.com/cilium/cilium/issues/17926#issue-1056743990 I read that this was discussed in one of our internal meetings?)

Reply:
Sorry your comment hadn't showed up yesterday :shrug:.
OK if we refer to the Clustermesh guide and if the latter has them, then it's all good for me.